deploy/charts/trust-manager/values.yaml

```
@@ -53,6 +53,17 @@ image:
 defaultPackage:
   # Whether to load the default trust package during pod initialization, and include it in main container args. This container enables the 'useDefaultCAs' source on Bundles.
   enabled: true
+  # Kubernetes pod resource limits for default package init container.
```

```suggestion
  # Kubernetes Pod resource requests and limits for the default trust package init container.  
```

The original text matches the documentation for the existing resources value, so happy if you prefer to keep it as-is for consistency

https://github.com/cert-manager/trust-manager/blob/f728840728dd599064c7583e797444b3872d007b/deploy/charts/trust-manager/values.yaml#L106-L116